include/boost/sml/aux_/utility.hpp

```diff
@@ -397,4 +415,8 @@ struct string<T> {
 
 }  // namespace aux
 
+template <typename T>
+auto wrap(T callback) {
+  return aux::zero_wrapper{callback};
```

Comment:
Seems to fail in the CI
https://travis-ci.org/github/boost-ext/sml/jobs/728453891